### PR TITLE
Add symmetric cryptography slides

### PR DESCRIPTION
**Pull request by Rothj**
# Description

* Added the initial version of the slides on symmetric cryptography
* Added two images: image_Enigma and drawing_key_exchange
* Added Simon Singh to the ref.bib file
* Added the LaTeX package colortbl to the config file at line 9. The package is freely available under the LATEX Project Public License (LPPL) and used to add colour in LaTeX tables.

## Type of Change

- [x] Major feature (New files or assets, large changes to files)

# Checklist:

- [x] I have performed a self-review of my own changes
- [x] I have made sure that any changed LaTeX files compile properly
- [x] I have assigned at least one reviewer

### CODE REVIEW

File: slides/Symmetric Cryptography.tex [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+	\end{table}
+	\uncover<3->{
+		\begin{tikzpicture}[overlay]
+			\draw[focus, line width=1.5pt] (2.04,2.1) ellipse (0.6cm and 0.75cm);
+			\draw[focus, line width=1.5pt] (5.42,2.1) ellipse (0.6cm and 0.75cm);
+			\draw[focus, line width=1.5pt] (8.4,2.1) ellipse (0.6cm and 0.75cm);
+		\end{tikzpicture}
+	}
+	\begin{itemize}
+		\item<2-> "the" can be enrcrypted in four ways: DPR, BUK, GNO, ZRM
```
Comment (dariothuerkauf) [inline slides/Symmetric Cryptography.tex:413]:
```suggestion
		\item<2-> using a code word with 4 letters, "the" can be encrypted in four ways: DPR, BUK, GNO, ZRM
```

File: slides/Symmetric Cryptography.tex [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+%%%
+
+%%%
+\begin{frame}{DES: Data Encryption Standard}
+	\begin{itemize}
+		\item<1-> Companies need a standardized approach
+		\item<2-> Encryption Method "Lucifer" by Horst Feistel in early 1970s.
+		\item<3-> Becomes "Data Encryption Standard (DES)"
+		\item<4-> Advantage: Standardisation and security
+		\item<5-> Disadvantag: Key distribution problem persists
```
Comment (dariothuerkauf) [inline slides/Symmetric Cryptography.tex:488]:
```suggestion
		\item<5-> Disadvantage: Key distribution problem persists
```

File: slides/Symmetric Cryptography.tex [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+				\caption{ASCII binary letters. Based on \cite{singh1999}}
+		\end{table}}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{DES: Data Encryption Standard}
+	\begin{itemize}
+		\item<1-> Companies need a standardized approach
+		\item<2-> Encryption Method "Lucifer" by Horst Feistel in early 1970s.
```
Comment (dariothuerkauf) [inline slides/Symmetric Cryptography.tex:485]:
```suggestion
		\item<2-> Encryption Method "Lucifer" by Horst Feistel in early 1970s
```

File: slides/Symmetric Cryptography.tex [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+			\draw[focus, line width=1.5pt] (2.04,2.1) ellipse (0.6cm and 0.75cm);
+			\draw[focus, line width=1.5pt] (5.42,2.1) ellipse (0.6cm and 0.75cm);
+			\draw[focus, line width=1.5pt] (8.4,2.1) ellipse (0.6cm and 0.75cm);
+		\end{tikzpicture}
+	}
+	\begin{itemize}
+		\item<2-> "the" can be enrcrypted in four ways: DPR, BUK, GNO, ZRM
+		\item<4-> This depends on its relative position to the code word
+		\item<5-> "the" is encrypted twice with BUK
+		\item<6-> Thanks to this we can gues the length of the code word
```
Comment (dariothuerkauf) [inline slides/Symmetric Cryptography.tex:416]:
```suggestion
		\item<6-> Thanks to this repetition the length of the code word can be guessed
```

**Comment by Chainomics** [comment]
@Rothj Great job! Did you make sure that the enigma picture has an open license? If yes, please remove the border, i.e. make everything around the machine transparent. Please also review my commit. I have fixed some typos, resolved some LaTex issues, and made some minor changes to the notation. 

Regarding the TeX issues:
* Commas in numbers should only be used outside math environments.
* Please use ```<text>''` instead of `"<text>"`.

File: slides/Symmetric Cryptography.tex (hunk 1 above)
Comment (Rothj) [reply slides/Symmetric Cryptography.tex]: I would like to keep the four alternatives how "the" can be encrypted in this example.

**Comment by Rothj** [comment]
I included the review of @dariothuerkauf  and made the background of the enigma image transparent.

The image of the enigma is from Wikipedia and there published under the creative commons license (CC BY-SA 4.0). In short, we are free to use and change it as long as we reference the source, which we do.

**Review by Chainomics — APPROVED** [review]
Thanks for the changes @Rothj! Nothing else to add.

**Pull request merged**